Dockerfile

```diff
@@ -1,6 +1,17 @@
+FROM golang:1.12 AS build-image
+
+ADD . /go/src/github.com/kubeflow/pytorch-operator
+
+WORKDIR /go/src/github.com/kubeflow/pytorch-operator
+
+# Build pytorch operator v1beta2 binary
+RUN go build github.com/kubeflow/pytorch-operator/cmd/pytorch-operator.v1beta2
```

Comment:
go build cmd/pytorch-operator.v1beta2
go build cmd/pytorch-operator.v1

Reply:
Yes,I agree with you:>